wdh_server/app.py

```diff
@@ -16,6 +16,9 @@
 store_location = os.path.dirname(os.path.abspath(__file__))
 STORE = os.path.join(store_location, "store.yml") # TODO: read from config
 
+SEARCH_CATEGORY_AUTHOR = "author"
+SEARCH_CATEGORY_ARTICLE = "article"
```

Comment:
ich bin kein Freund solcher Konstanten - in C o.Ä. macht das Sinn, in Python ist das meist unidiomatisch